docs/static/releasenotes.asciidoc

```diff
@@ -59,6 +59,14 @@ This section summarizes the changes in the following releases:
 [[logstash-7-16-1]]
 === Logstash 7.16.1 Release Notes
 
+[[known-issue-7-16-1]]
+==== Known issue
+
+Release 7.16.1 includes updated versions of filters that will not run on JDK 8.
+We are actively patching the build scripts for all filters that are affected. 
+
+Workaround: Clear the `JAVA_HOME` variable to use the JDK version (Java 11) bundled with Logstash. 
```

Comment:
I was going to suggest we should point to the [JAVA_HOME section](https://www.elastic.co/guide/en/logstash/current/getting-started-with-logstash.html#java-home) of the docs, but it looks like we don't tell users how to unset `JAVA_HOME` in the appropriate way for their OS...

So, may be refer to `JAVA_HOME` environment variable instead of variable?